production/terraform/modules/s3/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "us-east-2"
+}
+
+variable "bucket_name" {
+  description = "Bucket name for Loki storage"
+  type    = string
+  default = "loki-data"
+}
+
+variable "cluster_name" {
+  description = "Name of EKS cluster"
+  type = string
+}
+
+/*
+-var oidc_id="$(aws eks describe-cluster --name <cluster_name> --query 'cluster.identity.oidc.issuer' --output text | sed -e 's/^https:\/\///')"
```

Comment:
wondering if you could retrieve the oidc id from the aws_eks_cluster datasource instead of manually  feeding it as a variable

Reply:
maybe this could be in the description field of the variable instead of a comment

Reply:
Good catch. I meant to do this but gave up. Ah, I mixed datasource and resource https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/eks_cluster

Reply:
It worked. Nice!